libs/core/tests/unit_tests/utils/test_utils.py

```diff
@@ -111,10 +111,18 @@ def test_check_package_version(
             {"a": [{"idx": 0, "b": "f"}]},
             {"a": [{"idx": 0, "b": "{"}, {"idx": 0, "b": "f"}]},
         ),
+        # int replacing
+        (
+            {"index": 1, "b": "a"},
+            {"index": 5, "b": "b"},
+            {"index": 5, "b": "ab"},
```

Comment:
mm is this always the right behavior? if two things have different indexes then presumably they shouldn't be merged?

in general its not clear how to "merge" ints, which is why its not supported. certain ints should be added (eg incremental token counts), certain ints should be max/min-ed (eg timestamps), certain ones should be replaced (eg total token counts). there's no general way to handle ints

Reply:
This is good question. Very hard to say, but now the streaming pi with Bedrock fails due to this issue that index changes.

Reply:
I'm new here and don't udnerstand why CI fails (test-doc-imports skipped, perhaps normal) and Vercel deployment failed. I don't know what this means.